.ci/install_footloose.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o nounset
+set -o pipefail
+
+cidir=$(dirname "$0")
+arch=$("${cidir}"/kata-arch.sh -d)
+source "${cidir}/lib.sh"
+
+if [ "$arch" != "x86_64" ]; then
+	die "Footloose only available with x86_64."
+fi
+
+footloose_repo="github.com/weaveworks/footloose"
+
+GO111MODULE=on go get "$footloose_repo"
```

Comment:
OOI why is this being used as [we haven't switched yet](https://github.com/kata-containers/kata-containers/issues/42) :)

Reply:
https://github.com/weaveworks/footloose#from-source :smile:

Reply:
@jodh-intel , I am using the installation as it the documentation says

Reply:
Thanks.